File: remove-npmignore.js
const fs = require('fs');
const path = require('path');
const glob = require('glob');

// Use glob to find all .npmignore files in the libs directory
const pattern = path.join(__dirname, 'libs', '**', '.npmignore');
const files = glob.sync(pattern);

console.log(`Found ${files.length} .npmignore files to remove:`);

files.forEach((file, index) => {
  try {
    fs.unlinkSync(file);
    console.log(`${index + 1}. Removed: ${file}`);
  } catch (error) {
    console.error(`Error removing ${file}:`, error.message);
  }
});

console.log('✓ All .npmignore files have been removed from the libs directory.');

File: remove_npmignore.py
#!/usr/bin/env python3
import os
import glob

# Find all .npmignore files in the libs directory
npmignore_files = glob.glob('/Users/leenorris/github.com/onivoro/monorepo/libs/**/.npmignore', recursive=True)

# Remove each file
for file_path in npmignore_files:
    try:
        os.remove(file_path)
        print(f"Removed: {file_path}")
    except OSError as e:
        print(f"Error removing {file_path}: {e}")

print(f"Total files removed: {len(npmignore_files)}")